README.md

```diff
@@ -41,24 +41,20 @@ For `[]byte` fields the default encoding used for tree and proof generation is H
 See below code sample (`examples/simple.go`) for a usage example. For detailed usage, check godocs.
 
 ```go,
-  // ExampleDocument is a protobuf message
-	document := documentspb.ExampleDocument{
-		Value1:      1,
-		ValueA:      "Foo",
-		ValueB:      "Bar",
-		ValueBytes1: []byte("foobar"),
+  // ContainSalts is a protobuf message which contain salts message
+	document := documentspb.ContainSalts{
+		ValueA:     "valueAValue",
+		ValueB:     10,
+		Salts:      []*proofspb.Salt{{Compact: []byte{0,0,0,1}, Value: []byte{0x1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x2}},{Compact: []byte{0,0,0,2}, Value: []byte{0x3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0x4}}},
```

Comment:
We shouldn't have to set the salts here anymore, right? And could we use ExampleDocument again here?

Reply:
ok